lib/LedgerSMB/Database/Upgrade.pm

```diff
@@ -185,11 +185,15 @@ sub _linked_accounts {
 }
 
 sub _available_countries {
+    # user locale?
```

Comment:
What do you mean by this remark? Can you either elaborate or remove? (I mean, if there is a TODO here, please mention that. If there is a consideration not taken into account, but should be in the future, it's good to make that explicit.)

Reply:
I'd rather use the user's instead of Sysconfig. Just wondering what is the best for setup

Reply:
Setup should use the admin's preconfigured value from Sysconfig. However, libraries this deep in the call chain should be ambivalent. If we need a language, I guess we need an initarg to specify one.

Reply:
Fixed